tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
I disagree, Nils. From my perspective, the motivaiton behind #147 is that tsfresh was, due to the multiprocessing, spamming the command lines with no logger found messages in the line of 

```
No handlers could be found for logger "tsfresh.feature_selection.feature_selector"
No handlers could be found for logger "tsfresh.feature_selection.feature_selector"
No handlers could be found for logger "tsfresh.feature_selection.feature_selector"
No handlers could be found for logger "tsfresh.feature_selection.feature_selector"
...
```

I am fine with a singular message that no logger was found. However, having 10s or 100s of these messages is quite annoying. Now, if I get the changes right, we have no message at all. I think not saying anything about a missing logging backend is dangerous.